apps/src/StudioApp.js

```diff
@@ -944,6 +961,15 @@ StudioApp.prototype.toggleRunReset = function(button) {
 
   getStore().dispatch(setIsRunning(!showRun));
 
+  if (showRun) {
+    if (this.editDuringRunAlert !== undefined) {
+      ReactDOM.unmountComponentAtNode(this.editDuringRunAlert);
+      this.editDuringRunAlert = undefined;
+    }
+  } else {
+    this.executingCode = this.getCode().trim();
```

Comment:
Still needed?

Reply:
The change handlers are also run when you toggle between code and block mode in droplet so we need the code when the user clicked run to compare against later to see if there was an actual change.

Reply:
👍